Update rise-storage.html

woneal1983 opened the pull request:
I wanted to add this as a special request for windows IOT. The image folder and single image acts differently on windows IOT than it does on a chrome player because iot uses edge. Dont know why its different but seems i have to send different headers for both to work. This &single=true will allow me to know which request is coming in when the cachebuster is called. A single image will have this on there. While the image folder will not. Please test this for me on your players to be sure this works. If it does please merge this in so that i can complete the Windows IOT caching side of things. Thanks

stulees commented:
@woneal1983 We discussed your request and we agreed that the best thing we can do is to ignore checking `this.isLoaded` and just always request the file as so:

```
this._cacheUrl = this._baseCacheUrl + "cb=" + new Date().getTime() + "?url=" + this._fileUrl
```

This way it will be consistent for you and it does not impact any functionality on our side. Can you let us know if that will solve your issue? 

We will queue up this work soon and will inform you once complete. Thanks 
@alanclayton @donnapep @rodrigopavezi FYI

woneal1983 commented:
The problem is with headers. In the single image side on iot only works for me when im sending 302 Found with location. Because iot doesnt like File-URL with 200 OK. Doesnt work at all. On the image folder side i can send 200 OK with location header because of the fix we added here.
https://github.com/Rise-Vision/widget-image/blob/master/dist/components/rise-storage/rise-storage.html#L890

The reason i added the request with the fix is because i need to know which callback is coming. When doing your callback both are the same and so the headers have to be the same. Problem is i cant send the same headers without one or the other messing up or going blank. I need to separate the types so i can tell which callback is coming in so i can send the correct headers for one or the other. 

Now the way you have it works great on chrome and i do the same exact thing as you on my android app. IOT on the other hand uses edge so there are things there that doesnt correctly align with what chrome accepts on the header side of things. 

With that being said i need to know which callback is coming. The single or the folder. If that can happen then i can easily do the rest on my side. Hope this explains the reasoning.
